script/Registry.s.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.25;
+
+// Factories
+address constant CURVE_STABLE_SWAP_FACTORY = 0xbC0797015fcFc47d9C1856639CaE50D0e69FbEE8;
```

Comment:
would be nice to indicate with comments that these contract address are the ones live on Gnosis mainnet